src/Services/PatientService.php

```diff
@@ -52,16 +51,16 @@ public function validate($patient, $context, $id = null)
         
         $this->validator->context('insert', function (Validator $context) {
             $context->required('first_name')->lengthBetween(2, 30);
-            $context->required('fname')->lengthBetween(2, 255);
-            $context->required('lname')->lengthBetween(2, 255);
-            $context->required('sex')->lengthBetween(4, 30);
-            $context->required('DOB')->datetime('Y-m-d');
+            $context->required('fname', "First Name")->lengthBetween(2, 255);
+            $context->required('lname', 'Last Name')->lengthBetween(2, 255);
+            $context->required('sex', 'Gender')->lengthBetween(4, 30);
+            $context->required('DOB', 'Date of Birth')->datetime('Y-m-d');
         });
         
         $this->validator->context('update', function (Validator $context) {
             $context->copyContext('insert', function ($rules) {
                 foreach ($rules as $key => $chain) {
-                    $this->validator->optional($key);
+                    $chain->required(false);
```

Comment:
Not so confident about this. 
Lets see if https://github.com/particle-php/Validator/pull/195 it gets merged 🤔